bundle/jsky.app.ot/src/main/scala/jsky/app/ot/ags/BagsManager.scala

```diff
@@ -157,8 +157,13 @@ object BagsState {
 
     // Successful AGS lookup while running (and not edited).  Apply the update
     // and move to IdleState.
-    private[ags] override def succeed(results: Option[AgsStrategy.Selection]): StateTransition =
-      (IdleState(k, Some(hash)), BagsManager.applyAction(k, results))
+    private[ags] override def succeed(results: Option[AgsStrategy.Selection]): StateTransition = {
+      val h = results.fold(hash) { r =>
+        if (r.posAngle === ctx.getPositionAngle) hash
```

Comment:
Oh, I see now that `AlmostEqual` is limited to the `edu.gemini.spModel.core` test files.
It seems we should have an operation like this on `Angle`, seeing as there are multiple cases where it is necessary. Does anyone know if anything like this exists? I can't find it, and don't want to reimplement, but it might be worthwhile to have an operation defined in `Angle` that compares to a certain set precision, say `0.005`?